.gitignore

```diff
@@ -113,8 +111,11 @@ venv.bak/
 .vscode/
 .idea/
 
-# Paths file
-paths.yml
-
 # PEP 582; used by e.g. github.com/David-OConnor/pyflow
 __pypackages__/
+
+# Files generated by the notebooks
```

Comment:
Do we have any teardown functions to remove these files generated during testing/notebook execution?

Reply:
I don't think that we do, since it wasn't necessarily an issue when ran locally. We should add it for ReadTheDocs, though.